hasher-matcher-actioner/.devcontainer/devcontainer.json

```diff
@@ -1,25 +1,20 @@
 // For format details, see https://aka.ms/devcontainer.json.
```

Comment:
did you mean to update devcontainer.json.example or rename back to devcontainer.json?

Reply:
Yes, the renaming was intentional from `devcontainer.json.example` to `devcontainer.json`

Reply:
great, I prefer that option it just make the docs a tad confusing